SteamKit2/SteamKit2/Steam/Handlers/SteamGameServer/SteamGameServer.cs

```diff
@@ -161,7 +161,7 @@ public void LogOnAnonymous( uint appId = 0 )
 
             logon.Body.client_os_type = ( uint )Utils.GetOSType();
             logon.Body.game_server_app_id = ( int )appId;
-            logon.Body.machine_id = MachineIdUtils.GenerateMachineID();
+            logon.Body.machine_id = HardwareUtils.GenerateMachineID();
```

Comment:
I just nethooked steamcmd, and anonymous logins do not have machine_id set. And anonymous logins worked fine last night.